apps/api/src/server/utils/document-storage.js

```diff
@@ -0,0 +1,43 @@
+import config from '../config/config.js';
+
+/**
+ * @typedef {Object} DocumentInfo
```

Comment:
Would make sense to align this to the document db type, we have to map to this type but it isn't really necessary. There's also annoying changes like `document.guid -> GUID`